src/core.js

```diff
@@ -12,10 +12,9 @@ export const jsonsHeaders = ((array) => Array.from(
 ));
 
 export const jsons2arrays = (jsons, headers) => {
- headers = headers || jsonsHeaders(jsons);
- return [headers, ...jsons.map((object) =>
-  headers.map((header) =>
-   object[header] ? object[header] : ''))]
+  headers = headers || jsonsHeaders(jsons);
+  const data = jsons.map((object) => headers.map((header) => (header in object) ? object[header] : ''));
```

Comment:
I like it.. yes this is the best solution or also `object.hasOwnProperty(header)`